grid.html

```diff
@@ -14,87 +15,109 @@
 		<link href='http://fonts.googleapis.com/css?family=Economica:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<link rel="stylesheet" media="(max-width: 768px)" href="css/mobile.css" type="text/css" />
```

Comment:
When possible, use single quotes instead of double quotes. They make your file size smaller. Apply when possible.